apps/web/src/components/SearchAndRecent.tsx

```diff
@@ -0,0 +1,110 @@
+import { FormRow } from './FormRow';
```

Comment:
Let's delete this since it's not used

Reply:
We should name the component the same as the file name, so either change the file name to `SearchAndSort` or change the component name to `SearchAndRecent`.